lib/wwwtxt/page.rb

```diff
@@ -0,0 +1,30 @@
+module WwwTxt
+  class Page
+    attr_reader :title
+    attr_reader :body
+
+    def initialize(title, html_body, renderer=PlainTextRenderer)
+      @title = title
+      @file_name = clean_title(title)
+      @body = renderer.render_from_html(html_body)
+      @format = renderer::FORMAT
+    end
+
+    def save(path)
```

Comment:
this can eventually be moved to the concern of a Writer class. we might want to push to a web service instead of writing to file, for instance